src/test/java/io/specto/hoverfly/junit/core/HoverflyConfigTest.java

```diff
@@ -10,11 +11,12 @@
 
     @Test
     public void shouldHaveDefaultSettings() throws Exception {
-        HoverflyConfig configs = HoverflyConfig.configs();
+
+        HoverflyConfiguration configs = configs().build();
 
         assertThat(configs.getHost()).isEqualTo("localhost");
-        assertThat(configs.getAdminPort()).isEqualTo(0);
-        assertThat(configs.getProxyPort()).isEqualTo(0);
+        assertThat(configs.getAdminPort()).isGreaterThan(0);
```

Comment:
Nothing has changed here. It just the build() method does validation and assign random port, that's why I am changing assertion to greaterThan 0.